projects/ngrx.io/src/index.html

```diff
@@ -26,7 +26,7 @@
   <!-- -->
 
   <link rel="manifest" href="pwa-manifest.json">
-  <meta name="theme-color" content="#1976d2">
+  <meta name="theme-color" content="#a829c3">
   <meta name="apple-mobile-web-app-capable" content="yes">
   <meta name="apple-mobile-web-app-status-bar-style" content="translucent">
 
```

Comment:
Will you fix the `noscript` area below, and add the NgRx logo and remove the sub header?

Reply:
Yes I will do so tomorrow.